modules/hostdb.go

```diff
@@ -40,8 +40,9 @@ type HostSettings struct {
 // coins frozen in the host's announcement transaction.
 type HostEntry struct {
 	HostSettings
-	IPAddress NetAddress
-	Weight    types.Currency
+	IPAddress   NetAddress
+	Weight      types.Currency
+	Reliability types.Currency
```

Comment:
I'm averse to using the wrong type for the job. Currency is *almost* the right type for Weight, so I guess keeping that wouldn't be so bad. But Reliability definitely seems like it should be a fractional type, either a float or a big.Rat.